### DISCUSSION TITLE
Do "js exec" blocks still exist?

### DISCUSSION THREAD

**Discussion by wlach** [pngwn/MDsveX#221] [Q&A] [answered]
The documentation mention `js exec blocks`:

> Obviously you have access to values defined in YAML (namespaced under \`metadata\`) and anything defined in an fenced \`js exec\` block can be referenced directly.

https://github.com/pngwn/MDsveX/blob/master/packages/site/src/routes/_source.js#L54

However, these don't seem to work in the repl and I can't find any references to them in the source. Am I correct in thinking this was an old feature that was taken out?

**Comment by pngwn** [comment, marked as answer]
No, they were used in an older version when it wasn't possible to use all svelte syntax. `js exec` blocks just used to compile down to script elements which you can now use directly.

I'll update that documentation, must have slipped through when things changed.
